bpf/generictracer/beyla_ipc.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
Can we please not name this file as Beyla_IPC, maybe IPC_EBPF, so we don't have to redo this for OBI.